scripts/ci/docker-compose/keycloak/keycloak-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# We exit in case cd fails
+cd /opt/keycloak/bin/ || exit
+
+# Start Keycloak in the background
+./kc.sh start-dev --http-port=38080 &
```

Comment:
Could/should we be using the KC_HOSTNAME_PORT environment variable that you are creating in  [scripts/ci/docker-compose/integration-keycloak.yml](https://github.com/apache/airflow/pull/43240/files#diff-e06ba92f98115758298308aff17e8fa62301ef34023a1fcaf7e3b7a4f288f512) instead of hardcoding it in here so many times?

Reply:
Yeah, that's fair, I didn't want to spend too much time on this one. But then I ended up doing that anyway, so I can polish that up.